platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/ProblemSummaryData.java

```diff
@@ -14,10 +14,27 @@
  * limitations under the License.
  */
 
-package org.gradle.problems.internal;
+package org.gradle.api.problems.internal;
 
-import org.gradle.api.problems.internal.ProblemEmitter;
-import org.gradle.problems.buildtree.ProblemReporter;
+import org.gradle.api.problems.ProblemId;
 
-public interface ProblemReportCreator extends ProblemReporter, ProblemEmitter {
+/**
+ * Problem summary data
+ */
```

Comment:
Trivial comment is trivial. Does sanityCheck fail if you don't add this?
If so, there's a missing dot at the end of the sentence.

Reply:
That was part of me figuring out what is going on [here](https://gradle.slack.com/archives/CBSJ2GUTV/p1731947425228549). Trying to trigger the build again and getting a saner result.